benchmarks/django_simple/start

```diff
@@ -0,0 +1,66 @@
+#!/usr/bin/env python3
+import json
+import os
+import subprocess
+import sys
+import time
+import tempfile
+import urllib.request
+
+from tenacity import retry
+from tenacity import stop_after_attempt
+from tenacity import wait_fixed
+
+
+@retry(
+    wait=wait_fixed(5),
+    stop=stop_after_attempt(50),
+)
+def wait_for_app(url="http://localhost:5000"):
+    return urllib.request.urlopen(url)
+
+
+def run():
+    with open("meta.json", "r") as f:
+        meta = json.load(f)
+
+    if meta is None:
+        sys.stderr.write("sirun config error")
+        sys.exit(1)
+
+    output_dir = os.path.join("/artifacts", "output")
+    os.makedirs(output_dir, exist_ok=True)
+    sirun_output = os.path.join(output_dir, "sirun.ndjson")
+    with open(sirun_output, "w") as o:
+        # we could use sirun's own functionality to run all variants but since
+        # we need to capture k6 summary we have to split out sirun runs by
+        # variant
+        for variant in meta["variants"]:
+            print(f"variant={variant}")
+            env = os.environ.copy()
+            env["SIRUN_VARIANT"] = variant
+            sirunp = subprocess.Popen(["sirun", "meta.json"], stdout=subprocess.PIPE, env=env)
```

Comment:
we should check to be sure the sirun process is alive/healthy, otherwise we'll just size and `wait_for_app` for the full time.

maybe `wait_for_app` should a. check sirun process is still active b. check if the web app is up yet ?